### PR TITLE
update domain mapping

### PR DESCRIPTION
**Pull request by dannyroberts**
This will just need to reindex hqdomains elasticsearch index, which should only take a couple minutes.

### CODE REVIEW

File: corehq/pillows/mappings/domain_mapping.py [hunk 1]
```diff
@@ -192,6 +192,8 @@
                 'is_snapshot': {'type': 'boolean'},
                 'is_test': {'type': 'string'},
                 'is_starter_app': {'type': 'boolean'},
+                'last_modified': {'format': "yyyy-MM-dd||yyyy-MM-dd'T'HH:mm:ssZZ||yyyy-MM-dd'T'HH:mm:ss.SSSSSS||yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'||yyyy-MM-dd'T'HH:mm:ss'Z'||yyyy-MM-dd'T'HH:mm:ssZ||yyyy-MM-dd'T'HH:mm:ssZZ'Z'||yyyy-MM-dd'T'HH:mm:ss.SSSZZ||yyyy-MM-dd'T'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss.SSSSSS||mm/dd/yy' 'HH:mm:ss",
```
Comment (dimagimon) [inline corehq/pillows/mappings/domain_mapping.py]: E501 line too long (349 > 115 characters)

File: corehq/pillows/mappings/domain_mapping.py [hunk 2]
```diff
@@ -192,6 +192,8 @@
...
+                                  'type': 'date'},
```
Comment (dannyroberts) [inline corehq/pillows/mappings/domain_mapping.py]: Ran

```python
DOMAIN_MAPPING['properties']['date_created'] == DOMAIN_MAPPING['properties']['last_modified']
```

in manage shell and checked that [there really is a property on `Domain` called `last_modified](https://github.com/dimagi/commcare-hq/blob/ee7bed5c13fadabb5dd6641db555698997efbe59/corehq/apps/domain/models.py#L340)

**Comment by dannyroberts** [comment]
@czue, @emord this is for https://github.com/dimagi/commcare-hq/pull/6311/files#r38746484

**Comment by emord** [comment]
(ES doesn't get tested)

**Comment by czue** [comment]
nice, thanks

**Pull request merged**